tools/data_manipulation/xarray/xarray_mapplot.xml

```diff
@@ -0,0 +1,204 @@
+<tool id="xarray_mapplot" name="NetCDF xarray map plotting" version="0.18.2">
+    <description>Visualize netCDF variables on a geographical map</description>
+    <requirements>
+        <requirement type="package" version="3">python</requirement>
+        <requirement type="package" version="1.5.6">netcdf4</requirement>
+        <requirement type="package" version="0.18.2">xarray</requirement>
+        <requirement type="package" version="0.19.0">cartopy</requirement>
+        <requirement type="package" version="3.4.2">matplotlib</requirement>
+        <requirement type="package" version="1.2">cmcrameri</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir output_dir &&
+    HOME=`pwd`  &&
+    mkdir -p `pwd`/mlp_tmpdir &&
+    MPLCONFIGDIR=`pwd`/mlp_tmpdir &&
+    echo "Galaxy xarray version @VERSION@" > output_dir/version.txt &&
+    python '$__tool_directory__/xarray_mapplot.py' '$input' '$var'
+    #if $time_idx!=""
```

Comment:
you should maybe cast this to str() before comapre.

Reply:
I think I find a better way to select times (from the list of coordinates).